src/dotnet/APIView/ClientSPA/src/app/_models/codePanelModels.ts

```diff
@@ -86,5 +87,6 @@ export class CodePanelNodeMetaData {
     this.isNodeWithNoneDocDiffInDescendants = false;
     this.bottomTokenNodeIdHash = '';
     this.isProcessed = false;
+    this.relatedNodeIdHash = '';
```

Comment:
What is this supposed to be used for?

Reply:
It is used to identify if a line is related to another line at same level. For e.g. An attribute line is related to its class line so when class is not visible in node style view then it's related line also must be hidden.